modules/transport.js

```diff
@@ -188,37 +188,22 @@ __private.receiveSignature = function (query, cb) {
 __private.receiveTransactions = function (query, peer, extraLogMessage, cb) {
 	var transactions;
 
-	async.series({
-		validateSchema: function (seriesCb) {
-			library.schema.validate(query, schema.transactions, function (err) {
-				if (err) {
-					return setImmediate(seriesCb, 'Invalid transactions body');
-				} else {
-					return setImmediate(seriesCb);
-				}
-			});
-		},
-		receiveTransactions: function (seriesCb) {
-			transactions = query.transactions;
+	transactions = query.transactions;
 
-			async.eachSeries(transactions, function (transaction, eachSeriesCb) {
-				if (!transaction) {
-					return setImmediate(eachSeriesCb, 'Unable to process signature. Signature is undefined.');
-				}
-				transaction.bundled = true;
+	async.eachSeries(transactions, function (transaction, eachSeriesCb) {
+		if (!transaction) {
+			return setImmediate(eachSeriesCb, 'Unable to process signature. Signature is undefined.');
```

Comment:
This isnt a signature problem if the transaction isnt found.

Reply:
You are right, I'll update it. It was there previously, so I wasn't sure about the reason behind it.